tools/integration_tests/run.py

```diff
@@ -37,17 +37,20 @@ def terminate_process(process):
         )
 
     def start_server():
-        the_server_subprocess = subprocess.Popen(target_binary,
-                                                 shell=True,
-                                                 env=os.environ)
+        the_server_subprocess = subprocess.Popen(
+            target_binary, shell=True, env=os.environ
+        )
 
         # TODO we should change this to watch for a log message on startup
         # to know when to continue instead of sleeping for a fixed amount
         time.sleep(20)
 
         return the_server_subprocess
 
-    os.environ.setdefault('SYNC_MASTER_SECRET', 'secret0')
+    os.environ.setdefault("SYNC_MASTER_SECRET", "secret0")
+    os.environ.setdefault("SYNC_CORS_MAX_AGE", "555")
+    os.environ.setdefault("SYNC_CORS_ALLOWED_ORIGIN", "localhost")
```

Comment:
is this kinda jank to make integration tests feasible?

Reply:
Yep. Since we're creating a test server, doing this sort of thing is fine. We're just setting initialization parameters.